api/src/main/java/org/pmiops/workbench/notebooks/NotebooksConfig.java

```diff
@@ -0,0 +1,62 @@
+package org.pmiops.workbench.notebooks;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.logging.Logger;
+import org.pmiops.workbench.auth.UserAuthentication;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.exceptions.ServerErrorException;
+import org.pmiops.workbench.notebooks.api.ClusterApi;
+import org.pmiops.workbench.notebooks.api.NotebooksApi;
+import org.pmiops.workbench.notebooks.api.TestApi;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+@org.springframework.context.annotation.Configuration
+public class NotebooksConfig {
+
+  private static final Logger log = Logger.getLogger(NotebooksConfig.class.getName());
+
+  private static final String NOTEBOOKS_CLIENT = "notebooksApiClient";
+
+
+
+  @Bean(name=NOTEBOOKS_CLIENT)
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public ApiClient notebooksApiClient(UserAuthentication userAuthentication,
+      WorkbenchConfig workbenchConfig) {
+    ApiClient apiClient = new ApiClient();
+    // TODO: replace with whatever the actual creds for notebooks are
+    apiClient.setAccessToken(userAuthentication.getCredentials());
```

Comment:
Does using end user creds actually work? I thought you had to use the special SA creds that Hussein provided...

Reply:
So, I actually checked this, and it does work. The service account needs to be passed in to Leo in order to allow it to access firecloud functionality and endpoints, but we can auth for the API calls with any creds that have access to the dev environment.